src/services/MintingRegistrar.ts

```diff
@@ -13,44 +13,47 @@ export class MintingRegistrar {
   ) {}
 
   async maybeTryRegister() {
-    const lastCheck = await this.storage.getItem(
-      "minting_registrar/last_check",
-    );
+    const lastCheck = await this.storage.getItem(this.key("last_check"));
     const now = new Date().getTime() / 1000;
     const shouldCheck =
       lastCheck == null ||
       now > parseInt(lastCheck) + this.statusCheckSleepSeconds;
     if (!shouldCheck) return;
 
+    return await this.tryRegister(now);
+  }
+
+  async tryRegister(maybeNow?: number) {
+    const now = maybeNow ?? new Date().getTime() / 1000;
+
     try {
-      await withLock(this.storage, "minting_registrar/lock", async () => {
-        const protocol = await getProtocolService();
-        await protocol.ensureIdentityRegistered();
-        const isRegistered = await protocol.isRegisteredForMinting();
-        if (isRegistered) {
-          console.log(
-            "Already registered for next minting, not doing anything",
-          );
-        } else {
-          console.log("Not registered for minting, trying to register");
-          try {
+      await withLock(this.storage, this.key("lock"), async () => {
+        try {
+          const protocol = await getProtocolService();
+          await protocol.ensureIdentityRegistered();
+          const isRegistered = await protocol.isRegisteredForMinting();
+          if (isRegistered) {
+            console.log(
+              "Already registered for next minting, not doing anything",
```

Comment:
Remove the comma from the `console.log`

Reply:
This came from the formatter. So it would thrash or fail the build if formatting doesn't match.